src/components/panel/panel.js

```diff
@@ -945,13 +960,24 @@ MdPanelRef.prototype.show = function() {
   // TODO(ErinCoughlan) - Cancel any in-progress actions.
 
   var self = this;
-  this._showPromise = this._$q(function(resolve, reject) {
+
+  var animatePromise = function() {
     self.removeClass(MD_PANEL_HIDDEN);
-    self._animateOpen().then(function() {
-      self._focusOnOpen();
-      resolve(self);
-    }, reject);
-  });
+    return self._animateOpen();
+  }
+
+  this._showPromise = this._$q.all([
+      this._backdropRef ? this._backdropRef.show() : this._$q.resolve(self),
+      animatePromise().then(function() {
+        return self._$q.resolve(self);
```

Comment:
If I just used "self._$q.resolve" I wouldn't be able to specify the self param. I don't know if that strictly matters though or if the "then" passes it anyways.